app/controllers/find/v2/results_controller.rb

```diff
@@ -21,8 +21,10 @@ def search_courses_params
           :send_courses,
           :applications_open,
           :further_education,
+          :funding,
```

Comment:
This one sounds duplicate but not! It is to be backwards compatible with the currently production page. See commit message and PR description.